### PR TITLE
Splitted the special pages into different types, removed unused code

### PR DESCRIPTION
**Pull request by royderks**

### CODE REVIEW

File: src/app/pages/SpecialDetailPage/specials/Dashboard.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import setIframeSize from '../../../../shared/services/set-iframe-size/setIframeSize'
+import LoadingIndicator from '../../../../shared/components/loading-indicator/LoadingIndicator'
+
+const IFrameContainer = styled.div`
+  position: relative;
+  text-align: center;
+  width: 100%;
+
+  & iframe {
+    min-height: 80vh;
```
Comment (chrisvanmook) [inline src/app/pages/SpecialDetailPage/specials/Dashboard.jsx]: why the 80vh?

File: src/app/components/Video/__snapshots__/Video.test.jsx.snap [hunk 2]
```diff
@@ -3,6 +3,7 @@
 exports[`Video should render 1`] = `
```
Comment (ebabel-eu) [inline src/app/components/Video/__snapshots__/Video.test.jsx.snap]: Is that good to include snapshots into the repository?

File: src/app/pages/SpecialDetailPage/specials/Dashboard.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    setIframeLoading(false)
+
+    // Handle resize after the iframe is loaded
+    handleResize(setIframeHeight)
+  }
+
+  React.useEffect(() => {
+    window.addEventListener('resize', handleResize)
+
+    return function cleanup() {
```
Comment (chrisvanmook) [inline src/app/pages/SpecialDetailPage/specials/Dashboard.jsx]: why not just `return () => `?

File: src/app/components/Video/__snapshots__/Video.test.jsx.snap (hunk 2 above)
Comment (royderks) [reply src/app/components/Video/__snapshots__/Video.test.jsx.snap]: For new components we don't use snapshots any more, this however is an old component

**Pull request merged**